playbooks/roles/node-exporter/tasks/main.yml

```diff
@@ -10,3 +10,13 @@
     dest: /etc/consul/node-exporter.json
   notify:
     - reload consul
+
+- name: flush handlers
```

Comment:
@aayushagra Flushing handlers here will force all handlers to be run, instead of at the end of playbooks, which might, in certain cases, break deployments. 

Is there any reason why this is here?
If so, can you run specific actions on which handler you need run instead of flushing everything before the end of the playbooks?

Reply:
@giovannicimolin Actually i just did that because sometimes node-exporter gets disabled right after consul is reloaded. I guess it's an extra bug that i found and fixed but removing it should still allow the original code to function.

If flush handlers can cause problems it's best to remove it entirely so I'll do that